src/components/Web3Status/index.tsx

```diff
@@ -176,11 +181,11 @@ function Web3StatusInner() {
         {!hasPendingTransactions && connector && <StatusIcon connector={connector} />}
       </Web3StatusConnected>
     )
-  } else if (error) {
+  } else if (error || switchNetworkErrorId) {
```

Comment:
why do we show wrong network here? 
I think we dont need switchNetworkErrorId state, just show popup error is ok

Reply:
Don't know. I just follow the design.

Reply:
I discussed with Linh, He agrees that we can remove wrong network error, just show error popup is enough